spec.bs

```diff
@@ -1340,6 +1343,49 @@ Note: This is because we need to ensure that we do not leak <var ignore>creator<
 document's referrer|referrer=], [=Document/origin=], [=creator base url=], [=Document/policy
 container=], across the fenced frame boundary.
 
+<h3 id=local-policy-container-monkeypatch>Local Requests Policy Container Patch</h3>
+
+When making local requests, <{iframe}>s inherit their [=Document/policy container=] from their
```

Comment:
"local requests" is ambiguous. Do you mean "local requests" as in https://wicg.github.io/local-network-access/#local-network-request, since this is a PNA-focused PR, or do you mean requests whose scheme is local: https://fetch.spec.whatwg.org/#local-scheme. I thought the latter, but the terminology you chose (but didn't link to) is very reminiscent of the former, but inconsistent with your usage of "is local" below, which you fail to link to.

Reply:
I mean local scheme navigations. I'll update the section.